tests/query/stateless/test_lookup.py

```diff
@@ -33,97 +35,125 @@ def test_edge_index(self):
         resp = self.execute("CREATE EDGE INDEX like_index_1 on like(likeness)")
         self.check_resp_succeeded(resp)
         time.sleep(self.delay)
-        resp = self.execute('INSERT EDGE like(likeness) VALUES 100 -> 101:(95)')
+        resp = self.execute('INSERT EDGE like(likeness) VALUES "100" -> "101":(95)')
```

Comment:
In fact we will support the int vid, so I think better to copy the test file to `v2` folder and the modify.

Reply:
Good suggest. But I think it's better to put it in other PR once int VID is ready.

Reply:
Until to now, we don't modify the origin test cases and add new in `stateless/v1`, so I think we'd better not to change it.